### PR TITLE
11.Kubernetes.Basic.Objects.Full

### PR DESCRIPTION
**Pull request by Pro100chok91**

### CODE REVIEW

File: Kirill_Fadeev/11.Kubernetes.Basic.Objects/README.md [hunk 1]
```diff
@@ -0,0 +1,178 @@
+# Homework Assignment 1. Nginx deployment
+
+## Create deployment of nginx service:
+  * replicas: 4
+  * set resources for pods
+  * deployment shouldn't have any outage (service is available all time)
+  * ingress rule for host name (nginx-test.k8s-<NUMBER>.sa)
+  
+  * nginx.yaml
+```bash
```
Comment (pluhin) [inline Kirill_Fadeev/11.Kubernetes.Basic.Objects/README.md:10]: it is yaml, not bash

File: Kirill_Fadeev/11.Kubernetes.Basic.Objects/README.md [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+            memory: 200Mi
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: nginx-service
+  labels:
+    run: nginx-service
+spec:
+  type: LoadBalancer
```
Comment (pluhin) [inline Kirill_Fadeev/11.Kubernetes.Basic.Objects/README.md:53]: please remove, if you are not going to use loadbalancer

**Review by pluhin — CHANGES_REQUESTED** [review]

File: Kirill_Fadeev/11.Kubernetes.Basic.Objects/README.md (hunk 1 above)
Comment (Pro100chok91) [reply Kirill_Fadeev/11.Kubernetes.Basic.Objects/README.md]: fixed

**Comment by Pro100chok91** [comment]
Added Homework 12.

**Review by pluhin — APPROVED** [review]

**Review by pluhin — APPROVED** [review]

**Pull request merged**